examples/meta/src/distance/sparseeuclidean.sg.in

```diff
@@ -0,0 +1,20 @@
+CSVFile f_feats_a("@SHOGUN_DATA@/fm_train_sparsereal.dat")
+CSVFile f_feats_b("@SHOGUN_DATA@/fm_test_sparsereal.dat")
+
+#![create_features]
+Features features_a = features(f_feats_a)
+Features features_b = features(f_feats_b)
+#![create_features]
+
+#![create_instance]
+Distance d = distance("SparseEuclideanDistance", lhs=features_a, rhs=features_a)
```

Comment:
as the documentation stats SparseEuclideanDistance is expecting SparseFeatures.... imo the features you are reading above are not SparseFeatures but DenseFeatures.

plz make sure that you compile and test your changes before sending in a PR

Reply:
In all the other examples(`.sg.in` files), `features_a` and `features_b` are defined as variables of the `Features` class, and they directly become DenseFeatures by calling the constructor of the `CDenseFeatures` class. 
How should we ensure that the features are sparse and the constructor of the `CSparseFeatures` class is called?

Reply:
none of the examples are actually using methods that requires SparseFeatures... for that actually you'll need to patch factory.h to be able to read sparse format (say libsvmfile) and create a SparseFeature from it.

Reply:
That might be a bit beyond an entrance task so might be worth starting an issue and then rebase this when it’s done?

Reply:
Ah nvm! Does it work? :)

Reply:
Started new issue #4947